pom.xml

```diff
@@ -276,6 +276,12 @@
 			<version>1.2.17</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.data</groupId>
```

Comment:
I'd suggest using Spring Data Commons to enable retry functionality for all Spring Data modules.